src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -0,0 +1,85 @@
+#include "QueueNotifier.h"
```

Comment:
Style guide: includes should always contain the directory:

    #include "ngsiNotify/QueueNotifier.h"